pkg/restapi/vc/operation/operations_test.go

```diff
@@ -1089,12 +1081,12 @@ func TestStoreVCHandler(t *testing.T) {
 		rr := httptest.NewRecorder()
 		op.storeVCHandler(rr, req)
 		require.Equal(t, http.StatusBadRequest, rr.Code)
-		require.Equal(t, rr.Body.String(), errVaultNotFound.Error())
+		require.Equal(t, rr.Body.String(), "key is nil")
 	})
-	t.Run("store vc err vault not found", func(t *testing.T) {
```

Comment:
This test was a duplicate, so I removed it (the diff makes it look like I "replaced" it with another test, but in reality I removed the duplicate and added a test).